.papr.yml

```diff
@@ -67,16 +67,15 @@ timeout: 30m
 host:
   distro: fedora/25/atomic
 
-# mount yum.repos.d to get any injected repos from the host, which
+# copy yum.repos.d to get any injected repos from the host, which
 # will point to a closer mirror
 tests:
-  - >
-      docker run --privileged --rm
-      -e RPMOSTREE_COMPOSE_TEST_USE_HOST_REPO=1
-      -v /etc/yum.repos.d:/etc/yum.repos.d:ro
-      -v $(pwd):/srv/code
-      registry.fedoraproject.org/fedora:25 /bin/sh -c
-      "cd /srv/code && ./ci/build.sh && make install && ./tests/compose"
+  - docker run --privileged --rm -d --name composer
+    -e RPMOSTREE_COMPOSE_TEST_USE_HOST_REPO=1 -v $(pwd):/srv/code
+    registry.fedoraproject.org/fedora:25 sleep infinity
```

Comment:
I find using `--rm` with `exec sleep infinity` to be a pretty weird trick.  If things fail the container is actually going to stick around.  I guess if we're doing this I'd add `docker kill composer` at the end of the run at least?

But it seems easier to me to do the ro-mount to a different dir and do the copying inside the container.